content/tutorials/token-authentication.textile

```diff
@@ -390,28 +398,28 @@ blang[swift].
 
 h2. Step 4 - Respond to authentication requests
 
-Before we move onto the client, let's first make sure the web server is capable of issuing token request to clients that need to authenticate with Ably. However, instead of jumping into code, lets quickly talk about how token authentication works and what the difference is between a "@Token@":https://www.ably.io/documentation/realtime/authentication#token-details and "@TokenRequest@":https://www.ably.io/documentation/realtime/authentication#token-request is.
+Before you move onto the client, you'll need to first make sure the web server is capable of issuing token requests to clients that need to authenticate with Ably. However, instead of jumping into code, let's quickly talk about how token authentication works and what the difference is between a "@Token@":https://www.ably.io/documentation/realtime/authentication#token-details and "@TokenRequest@":https://www.ably.io/documentation/realtime/authentication#token-request is.
```

Comment:
Retaining "we" would make more sense IMO, while still keeping the rest of the tone as 'you'. Moving onto something is what we are asking them to do while they are following along in the tutorial. wdyt?